### ISSUE TITLE
Pasting from VS Code escapes data

### ISSUE THREAD

**Issue by learn-more** [Zettlr/Zettlr#4027]
### Description

When pasting code from visual studio code (.py file, crlf newlines) into a zettlr code block, there are some unwanted transformations applied:

Each newline is converted into a double newline
Some characters are (html) escaped:

The .py file used for testing is the python file from this PR: https://github.com/ramensoftware/windhawk-mods/pull/80

![image](https://user-images.githubusercontent.com/610685/211415848-5b32ea67-85ef-45ea-81f7-c4dca43ebaf0.png)

### Reproducing

Open a code block (both python or c worked),
open the file from the PR in visual studio code,
paste the code into the code block.

note: Opening the same file in notepad++ does not apply the transformations!

### Zettlr Version

Beta (if applicable)

### Specify version

3.0.0- beta

### Installation Method

From the Website or GitHub

### Your Platform

- [X] Windows
- [ ] macOS
- [ ] Linux

### Architecture

- [X] Intel 64bit
- [ ] ARM 64bit

### Operating System Version

Windows 10 22H2 (Build 19045.2364)

### Additional Information

Educated guess based on the clipboard formats available is that zettlr tries to grab the html format that vs code provides:

VS Code copy:
![image](https://user-images.githubusercontent.com/610685/211416579-a28b21f9-5ba2-4b55-90f3-7daf8ae28dbb.png)

Notepad++ copy:
![image](https://user-images.githubusercontent.com/610685/211416602-858ebc13-08a9-4c27-b614-210cd7ac9460.png)

**Comment by nathanlesage** [comment]
This is correct and expected behavior. If an app provides HTML, Zettlr needs to assume that the contents contains more information than the plain text, which all applications provide as a fallback.

The idea is that, if you copy, e.g., from Office or from websites, most HTML code can be converted to Markdown. This only breaks down for VS Code (or other code editors). If we were to change that, users would have to jump through hoops to copy valid and well-formed Markdown from websites.

Honestly, I don't know why VS Code thinks it should provide code as HTML instead of relying on third-party software to understand how to deal with code.

To solve this, simply select "paste as plain text", or press `Cmd/Ctrl+Shift+V`.
